src/markdown-pages/docs/managers.md

```diff
@@ -5,64 +5,62 @@ title: "Managers"
+| **[Selenium Manager](/docs/managers/selenium-manager)**<br> ![alpha](../../images/alpha.svg)                     | Allows tests to drive Web Browser testing using Selenium.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| **z/OS 3270 Manager**<br> ![alpha](../../images/alpha.svg)                                                       | Provides tests and Managers with a 3270 client.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| **[z/OS Batch z/OS MF Manager](/docs/managers/zos-batch-zos-mf-manager)**<br> ![beta](../../images/beta.svg)     | Provides the default implementation of the z/OS Batch Manager using z/OS MF. Can only be used via the z/OS Batch Manager interface.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
This looks a little strange when rendered, almost as if two managers have become concatenated in the same row.  can we just call then z/OS Batch Manager rather than z/OS Batch Manager z/OS MF Manager?